src/libs/antares/study/parts/common/series.h

```diff
@@ -38,7 +38,7 @@ class TimeSeries
 {
 public:
     using TSNumbers = Matrix<uint32_t>;
-    using TScoefficients = Matrix<double, int32_t>;
+    using TS = Matrix<double>;
```

Comment:
Use this wherever `Matrix<double>` is used

Also, `double` is the default value for this template argument, so `Matrix<>` should be enough.